templates/zerver/help/add-a-link-preview.md

```diff
@@ -0,0 +1,28 @@
+# Add a link preview
+
+{!server-admin-only.md!}
+
+Link previews are a way to see an overview of the a link's content before
```

Comment:
Change `overview of the a link's` to `overview of a link's`. :)

Reply:
Ah good call, thanks!